### PR TITLE
Scrubs all mentions of Vox from the code 

### PR DESCRIPTION
**Pull request by alsoandanswer**
I hope I don't break anything

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/__defines/misc.dm [hunk 1]
```diff
@@ -70,13 +70,12 @@
 #define ASFX_AMBIENCE	1
 #define ASFX_FOOTSTEPS	2
 #define ASFX_VOTE		4
-#define ASFX_VOX		8
```
Comment (Alberyk) [inline code/__defines/misc.dm:73]: This is not the same vox as the species

File: code/defines/procs/announce.dm [hunk 2]
```diff
@@ -58,8 +58,6 @@
 				to_chat(M, "<span class='alert'>[message]</span>")
 				if (announcer)
 					to_chat(M, "<span class='alert'> -[html_encode(announcer)]</span>")
-				if(message_sound && !isdeaf(M) && (M.client.prefs.asfx_togs & ASFX_VOX))
```
Comment (Alberyk) [inline code/defines/procs/announce.dm:61]: This is not the same vox as the species

File: code/modules/client/preferences_ambience.dm [hunk 3]
```diff
@@ -2,7 +2,6 @@
 //	/client/proc/Toggle_asfx,
 //	/client/proc/Toggle_footsteps,
 	/client/proc/Toggle_asfx_vote,
-	/client/proc/toggle_vox_voice,
```
Comment (Alberyk) [inline code/modules/client/preferences_ambience.dm:5]: This is not the same vox as the species

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 4]
```diff
@@ -164,12 +164,6 @@
 	breathe_mul = 2
 	var/strength = 6
 
-/datum/reagent/dexalin/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
```
Comment (Alberyk) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm:167]: Only the vox check should be removed, the rest of the proc should stay.

**Review by Alberyk — CHANGES_REQUESTED** [review]

File: maps/aurora/aurora-1_centcomm.dmm [hunk 5]
```diff
@@ -56781,7 +56781,7 @@ aaM
 aaM
 aaM
 tZx
-vox
+raider
```
Comment (alsoandanswer) [inline maps/aurora/aurora-1_centcomm.dmm:56784]:
```suggestion
vox
```

Comment (alsoandanswer) [inline maps/aurora/aurora-1_centcomm.dmm:56784]:
```suggestion
vox
```

Comment (alsoandanswer) [inline maps/aurora/aurora-1_centcomm.dmm:56784]:
```suggestion
vox
```

**Pull request closed without merging**